app/src/main/res/layout/activity_main.xml

```diff
@@ -2,73 +2,11 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<!-- Ignore MergeRootFrame: we need a single container View where we can attach fragments, so
-     we definitely need to keep this FrameLayout (i.e. we can't <merge> here). -->
-<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/drawer_layout"
-    tools:ignore="MergeRootFrame"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:id="@+id/container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fitsSystemWindows="true"
-    tools:openDrawer="start">
+    android:focusable="true">
+</LinearLayout>
 
-    <RelativeLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-        <!-- This view must be specified first so that it's focused first when other views (e.g. menu drawer)
```

Comment:
Which reminds me, we should test a11y functionality with the overlay too (maybe file a follow-up to verify good behavior?)